test/unit/environment.test.js

```diff
@@ -259,11 +257,43 @@ describe('the environment scraper', function () {
 
   describe('with symlinks', function () {
     const nmod = path.resolve(__dirname, '../helpers/node_modules')
-
-    beforeEach(function (done) {
-      if (!fs.existsSync(nmod)) {
-        fs.mkdirSync(nmod)
+    const makeDir = async (dirp, mkdirDb) => {
+      const code = await fs
+        .mkdir(dirp)
+        .then(() => null)
+        .catch((err) => {
+          if (err.code !== 'EEXIST') {
+            return err
+          }
+          return null
+        })
+      // vestigial--mkdirDb is the async callback, invisibly appended to arguments when apply is used
+      if (mkdirDb) {
```

Comment:
I could see removing this conditional block entirely, along with the `mkdirDb` property.